### PR TITLE
fix(buttons): highlighting button properly on focus

### PR DESCRIPTION
**Pull request by msorianoBox**
Buttons didn't receive proper focus behaviour, and looked
image

In order to highlight correctly I used the hover behavior so the focus outline is noticeable
image

### CODE REVIEW

File: src/styles/common/_buttons.scss [hunk 1]
```diff
@@ -121,7 +121,8 @@ button svg {
     &:not(.bdl-is-disabled),
     &:not(.is-disabled) {
         &:focus {
-            border-color: $primary-color;
+            background-color: lighten($primary-color, 8%);
+            border: 2px solid $primary-color;
```
Comment (alexkrolick) [inline src/styles/common/_buttons.scss:125]: If the border width changes on focus, won't it cause layout shifts?

Comment (msorianoBox) [reply src/styles/common/_buttons.scss:125]: I'll switch it back to 1px :D

Comment (alexkrolick) [reply src/styles/common/_buttons.scss:125]: Even with 1px, the default is 0, so it will still shift.

**Comment by msorianoBox** [comment]
Buttons didn't receive proper focus behaviour, and looked 
Screen Shot 2021-08-24 at 13 06 21

In order to highlight correctly I used the hover behavior so the focus outline is noticeable
Screen Shot 2021-08-24 at 13 06 36

**Review by jasminevle — APPROVED** [review]

**Pull request merged**